templates/vars.yml.j2

```diff
@@ -181,3 +181,6 @@ matrix_nginx_proxy_proxy_synapse_additional_server_configuration_blocks:
   auth_basic "protected";
   auth_basic_user_file /nginx-data/matrix-synapse-metrics-htpasswd;
   }'
+
+
+matrix_synapse_container_extra_arguments: ['-m=2g']
```

Comment:
Can we make this 2.5 or even 3 knowing that mainly 3 workers will take up the whole spectrum (generics), master and nginx proxy will take another big part and the rest is much lower?